package-system/Qt/install_windows_extras.py

```diff
@@ -18,25 +18,37 @@
     "icu": "https://github.com/unicode-org/icu/releases/download/release-65-1/icu4c-65_1-Win64-MSVC2017.zip"
 }
 
-for name in dependencies:
-    print(f"Attempting to install {name}")
+current_dir = os.getcwd()
 
-    installer_link = dependencies[name]
-    file_name = os.path.basename(installer_link)
+try:
 
-    # Download the zip (if we haven't already, so that this script can be run iteratively)
-    if not os.path.exists(file_name):
-        print(f"Downloading {name} from {installer_link}")
+    # Set the current working directory to temp
+    current_dir = os.getcwd()
+    os.chdir("temp")
```

Comment:
Nit: Any reason to do this over tempfile.mkdtemp() ?

Reply:
The build script works off of a temp subdirectory and everything is done there. We could use a local temp dir with cleanup, but having a fixed temp folder makes developing the scripts much easier (for incremental reasons).